### PR TITLE
0.67.2 and gym support

### PR DESCRIPTION
**Pull request by goedzo**
Adds basic gym support. NO fighting. NO berrie feeding.

### CODE REVIEW

File: configs/config.json.example [hunk 1]
```diff
@@ -465,6 +465,17 @@
             }
         },
         {
+            "type": "GymPokemon",
+            "config": {
+                "enabled": false,
+                "feed_berries": false,
```
Comment (davidakachaos) [inline configs/config.json.example]: No need for this anymore in the config

File: pokemongo_bot/event_handlers/logging_handler.py [hunk 2]
```diff
@@ -130,8 +130,10 @@ class LoggingHandler(EventHandler):
         'catch_limit_on':                    'yellow',
         'catch_limit_off':                   'green',
         'revived_pokemon':                   'green',
-        'healing_pokemon':                   'green'
-        
+        'healing_pokemon':                   'green',
+        'deployed_pokemon':                  'green',
+        'gym_error':                         'red',
+        'fed_pokemon':                       'white'
```
Comment (davidakachaos) [inline pokemongo_bot/event_handlers/logging_handler.py]: Removed feeding, so this can go aswel

**Comment by davidakachaos** [comment]
@MerlionRock @solderzzc Things look good to me, agreed?

**Comment by MerlionRock** [comment]
@davidakachaos @solderzzc Looks good 👍

**Comment by davidakachaos** [comment]
👍

**Pull request merged**